.github/workflows/build-ci-docker-images.yml

```diff
@@ -0,0 +1,54 @@
+name: Build pr ci-docker
+
+on:
+  push:
+    branches:
+      - change-ci # for now let's only build on this branch
+  repository_dispatch:
+  workflow_call:
+    inputs:
+      image_postfix:
+        required: true
+        type: string
+  schedule:
+    - cron: "17 0 * * *"
+
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    if: ${{ contains(github.event.head_commit.message, '[push-ci-image]') && !cancelled() && (success() || failure()) }}
```

Comment:
As seen with you, TODO: handle on `main`

Reply:
That is here to check the status of the previous workflow run. We see if it 's needed